tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md

```diff
@@ -0,0 +1,69 @@
+---
+author_name: Christian Savchev
+author_profile: https://github.com/christiansavchev  
+title: Create an Action of a Type Email
+description: Learn how to create an action of a type email
+keywords: tutorial
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product-function>sap-btp-cockpit ]
+primary_tag: software-product>sap-btp--cloud-foundry-environment
+parser: v2
+---
+
+# Create an Action of a Type Email
```

Comment:
It is already in the meta data. Please remove.

Reply:
It is better to keep the title and description in the body of the md file and remove it from the meta data.